processor/src/main/resources/org/mapstruct/ap/internal/model/BeanMappingMethod.ftl

```diff
@@ -78,7 +78,13 @@
     </#list>
     <#if returnType.name != "void">
 
-    return ${resultName};
+    <#if resultType.builderType??>
+        return ${resultName}.build();
```

Comment:
did we find other examples than `build()`

Reply:
Check the new version, this is outdated

Reply:
sorry.. I did review the commits one-by-one..